sample-app/src/main/java/com/joinforage/android/example/ui/pos/POSFragment.kt

```diff
@@ -43,10 +46,13 @@ class POSFragment : Fragment() {
             object : ResultCallback<DeviceManager> {
                 override fun onFinish(deviceManager: DeviceManager) {
                     Log.i("CPay SDK", "DeviceManager created successfully")
+                    val terminalId = deviceManager.systemDevice.getSystemInfo(SystemDevice.SystemInfoType.IMEI)
+                    posViewModel.setTerminalId(terminalId)
```

Comment:
I'm not super confident in this. I'm not sure what ID we want to use and whether this is the right one.  @devinmorgan would need to verify this on an actual terminal.